games-board/gtkatlantic/gtkatlantic-0.6.3.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools xdg-utils
+
+DESCRIPTION="Monopoly-like game that works with the monopd server"
+HOMEPAGE="http://gtkatlantic.gradator.net/"
+SRC_URI="http://download.tuxfamily.org/gtkatlantic/downloads/v0.6/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-libs/libxml2
+	media-libs/libpng:0=
+	x11-libs/gtk+:3"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+pkg_postinst() {
+	xdg_icon_cache_update
```

Comment:
We can avoid this entirely by using the xdg eclass. It has [magic](https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/xdg.eclass?id=e543d9d8a52e29dd939f425ba6d947754f4b59b4#n46) to call the correct functions.